src/main/java/com/vackosar/gitflowincrementalbuild/control/ChangedProjects.java

```diff
@@ -5,30 +5,35 @@
 import org.codehaus.plexus.logging.Logger;
 import org.eclipse.jgit.api.errors.GitAPIException;
 
+import com.vackosar.gitflowincrementalbuild.boundary.Configuration;
+
 import javax.inject.Inject;
+import javax.inject.Named;
 import javax.inject.Singleton;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 @Singleton
+@Named("gib.changedProjects")
```

Comment:
do u have to use the named annot?

Reply:
That's what the manual says: https://maven.apache.org/maven-jsr330.html ("How to use JSR-330")

`plexus-component-metadata` plugin (successor of `sisu-maven-plugin`?) generates a file with all the `Named` beans which is then picked up by Maven.

The actual value of `@Named` is not important but it felt weird to not define a name value.

Reply:
Strange that you have to. In Guice you can do without custom names as default is class name.